integration-tests/tests/billing-projects.js

```diff
@@ -31,26 +28,10 @@ const billingProjectsPage = (testPage, testUrl) => {
       await click(testPage, clickable({ text: 'Spend report' }));
     },
 
-    selectMembers: async () => {
-      await click(testPage, clickable({ text: 'Members' }));
-    },
-
-    selectOwners: async () => {
-      await click(testPage, clickable({ text: 'Owners' }));
-    },
-
     selectProject: (billingProjectName) => {
       return click(testPage, clickable({ textContains: billingProjectName }));
     },
 
-    selectDeleteProjectMenuOption: async (projectName) => {
```

Comment:
I didn't touch the billing project deletion code as it was already in TypeScript. But I verified we already created unit test coverage that is more extensive than this test was.